v2/cmd/cve-annotate/main.go

```diff
@@ -74,12 +74,19 @@ var (
 const maxReferenceCount = 5
 
 // dead sites to skip for references
-var badrefsRegexes = []*regexp.Regexp{
-	regexp.MustCompile(`(www\.|\/)secunia\.com\/`),
-	regexp.MustCompile(`(www\.|\/)osvdb\.org\/`),
-	regexp.MustCompile(`(www\.|\/)securityfocus\.com\/`),
-	regexp.MustCompile(`(www\.|\/)iss\.net\/`),
-	regexp.MustCompile(`\/archives\.neohapsis\.com\/`),
+var badRefs = []string{
+	"/osvdb.org/",
```

Comment:
What about removing `/` and using the domain name directly (e.g. `securityfocus.com`)?

Reply:
While I think the chances of missing a site because of an overmatch are generally low, I'd still prefer to have some bounds on it, especially with "iss.net" which could easily overmatch on things.